src/GCodePathConfig.h

```diff
@@ -77,6 +80,12 @@ class GCodePathConfig
 
     double getFlowPercentage() const;
 
+    const SpeedDerivatives& getSpeedDerivatives() const;
```

Comment:
This function is unused when you follow on my comments below.
In that case remove this function please.